storage/SystemInfo/CmdBtrfs.cc

```diff
@@ -81,7 +85,11 @@ namespace storage
 		y2mil( "uuid line:" << *it );
 
 		Entry entry;
-		entry.uuid = extractNthWord( 3, *it );
+
+		if (!regex_search(*it, match, uuid_regex))
```

Comment:
if I get code this can only happen when line contain ` uuid: ` but not UUID_REGEX after it. So malformed output from btrfs?

Reply:
Yes, if the regex is not found it looks like a wrong output from btrfs (or the output has changed on purpose).